CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v0.2.1
+
+### Bug Fixes
+
+- Pin controller-runtime version to v0.1.4 to fix dependency issues
```

Comment:
Link to the PR #831 
That also includes a fix that locks the ansible Dockerfile's python idna package to 2.7.